nix/passthru-vendored.nix

```diff
@@ -6,23 +6,25 @@
   # This could be done much more elegantly if `buildRustPackage` supported
   # finalAttrs. When https://github.com/NixOS/nixpkgs/pull/194475 lands, we can
   # most likely get rid of this.
-  rust = package: { cargo-cyclonedx }: package.overrideAttrs
+  rust = package: { cargo-cyclonedx, name ? "", lib, stdenv }: package.overrideAttrs
```

Comment:
If we remove the name parameter from here for now, I'm happy to merge the PR. Then most of #109 is done already.

I want to think about how to fix the workspace issue a little more. Maybe there is a more elegant solution.